src/session.ts

```diff
@@ -219,6 +219,8 @@ async function updateSession(
       entitlements,
     } = decodeJwt<AccessToken>(accessToken);
 
+    options.onSessionRefreshSuccess?.({ accessToken, user, impersonator });
```

Comment:
suggestion: I'm wondering if it would be beneficial to expose `organizationId` here, too?

Reply:
I'll add this as a fast-follow